Communication/Packets/Incoming/Groups/ModerateThreadMessageEvent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Plus.Communication.Packets.Outgoing.Groups;
+using Plus.Communication.Packets.Outgoing.Rooms.Notifications;
+using Plus.HabboHotel.GameClients;
+using Plus.HabboHotel.Groups.Forums;
+using System.Threading.Tasks;
+
+namespace Plus.Communication.Packets.Incoming.Groups;
+
+internal class ModerateThreadMessageEvent : IPacketEvent
+{
+    public async Task Parse(GameClient Session, ClientPacket Packet)
+    {
+        var int1 = Packet.PopInt();
+        var int2 = Packet.PopInt();
+        var int3 = Packet.PopInt();
+
+        GroupForum forum = PlusEnvironment.GetGame().GetGroupForumManager().GetForum(int1);
+
+        if (forum is null)
+        {
+            Session.SendWhisper(("Uh-oh. The requested forum cannot be found."));
```

Comment:
If forum is null, return so the rest is not executed. Then you can drop the `?` from `forum?.`